src/clientcommands.cpp

```diff
@@ -106,6 +108,63 @@ void ClientCommands::sync()
   download( lp.projectNamespace, lp.projectName );
 }
 
+static void printVal( bool isJsonFormat,
+                      QTextStream &out,
+                      const QString &key,
+                      const QString &val,
+                      bool isLast )
+{
+
+  QChar separator;
+  QChar quote;
+  QString space;
+  if ( isJsonFormat )
+  {
+    separator = ',';
+    quote = '"';
+  }
+  else
+  {
+    separator = '\n';
+    space = " ";
+  }
+
+  out << quote << key << quote << ":"  << space << quote << val << quote
+      ;
+
+  if ( !isLast )
+    out << separator;
+}
+
+void ClientCommands::info( bool isJsonFormat )
+{
+  Q_ASSERT( isAuthorized() );
```

Comment:
1. why do we need to check if user is authorized?
2. why assert? better to output a normal error message (probably different syntax whether using JSON or not)